README.md

```diff
@@ -11,13 +13,82 @@ This tells you if your jobs are done. And stuff like that.
  - Gets input / output dataset names
  - Works with piping, other unix nice things
 
-How I do dat?
--------------
+## Installation
 
- 1. Clone me
- 2. Add the directory to your `PATH`
- 3. Type `pandamon [user.<your user name>]`
- 4. **Pretty colors!!!**
+### Install from PyPI
+
+You can install [`pandamonium` from PyPI][pandamonium_PyPI] into any Python
+virtual environment by simply running
+
+```
+python -m pip install pandamonium
+```
+
+### Install the old style of global scripts
+
+If you want to install the original version of `pandamonium` before it became a
+library and was a set of global level Python scripts you can still do that.
+
+1. Clone the repository at tag [`v0.1`][tag_v0.1]
+```
+git clone git@github.com:dguest/pandamonium.git --branch v0.1
+```
+2. Add the directory to your `PATH`.
+Maybe with something like the following in your `.bashrc`
+
+```
+# Add pandamonium to PATH
+if [ -d "your/path/stuff/goes/here/pandamonium" ]; then
+    PATH="your/path/stuff/goes/here/pandamonium${PATH:+:${PATH}}"
+    export PATH
+fi
+```
+
+[pandamonium_PyPI]: https://pypi.org/project/pandamonium/
+[tag_v0.1]: https://github.com/dguest/pandamonium/releases/tag/v0.1
+
+### Notes if working on a remote server
+
+If you are working from a remote server where you do not have control over your
+Python runtimes (e.g. LXPLUS, ALTAS Connect login nodes) it is recommended that
+you bootstrap `virtualenv` and a default Python virtual environment by adding
+the following to your `.bashrc` or `.bashrc_user`
+
+```
+# Ensure local virtualenv setup
+if [ ! -f "${HOME}/opt/venv/bin/virtualenv" ]; then
+    curl -sL --location --output /tmp/virtualenv.pyz https://bootstrap.pypa.io/virtualenv.pyz
+    python /tmp/virtualenv.pyz ~/opt/venv # Change this to python3 if available
+    ~/opt/venv/bin/pip install --upgrade pip
+    ~/opt/venv/bin/pip install virtualenv
+    mkdir -p ~/bin  # Ensure exists if new machine
+    ln -s ~/opt/venv/bin/virtualenv ~/bin/virtualenv
+fi
+
+# default venv from `virtualenv "${HOME}/.venvs/base"`
+if [ -d "${HOME}/.venvs/base" ]; then
+    source "${HOME}/.venvs/base/bin/activate"
+fi
+```
+
+After that source your `.profile` or `.bash_profile` and then if you want to
+create a default Python virtual environment run
+
+```
+virtualenv "${HOME}/.venvs/base"
+```
+
+You will now be dropped into a virtual environment named `base` each time you login.
+The virtual environment is not special in anyway, so you should treat it as you
+would any other.
```

Comment:
I would put this after "use", and I think it's more appropriately called "working with virtualenv" or something like that. If someone has never used a venv this is a lot to dump on them, and if they have I guess they don't need this information so early on.